src/components/application_manager/rpc_plugins/vehicle_info_plugin/include/vehicle_info_plugin/vehicle_info_plugin.h

```diff
@@ -72,6 +72,7 @@ class VehicleInfoPlugin : public plugins::RPCPlugin {
                                      VehicleInfoAppExtension& ext);
 
  private:
+  void UnsubscribeFromNotExistingVDItems();
```

Comment:
@AKalinich-Luxoft `...NonExisting...`

Reply:
@AByzhynar fixed in https://github.com/smartdevicelink/sdl_core/commit/334284c3bf4d0930bb84a16cb67d040b59102065